action/hook/hook.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package hook
+
+// Config represents the configuration necessary
+// to perform hook related quests with Vela.
```

Comment:
```suggestion
// to perform hook related requests with Vela.
```

the quests are back.. i think i missed this on another PR, too.

Reply:
Doh! Apologies 😅 

What can I say? I really like your feedback of quests sounding cooler